fmt.sh

```diff
@@ -88,6 +89,12 @@ USAGE
   esac
 done
 
+if [[ $is_test = 1 ]]; then
+  prettier_args="--check"
+else
+  prettier_args="--write"
+fi
+
```

Comment:
Move this up where all other `--test` check are run, we want to keep them together so that we can act on all of them at once if needed. Look at how this is done currently and replicate the same approach.